<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OknoGlowne</class>
 <widget class="QMainWindow" name="OknoGlowne">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>[*]BTEdytorMap</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/ikona.ico</normaloff>:/ikona.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="1">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>734</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionNowaPlansza"/>
    <addaction name="actionWczytaj"/>
    <addaction name="actionZapisz"/>
    <addaction name="actionZapiszJako"/>
    <addaction name="separator"/>
    <addaction name="actionWyjdz"/>
   </widget>
   <widget class="QMenu" name="menuWidok">
    <property name="title">
     <string>Widok</string>
    </property>
    <addaction name="actionPrzybliz"/>
    <addaction name="actionOddal"/>
   </widget>
   <widget class="QMenu" name="menuNarzedzia">
    <property name="title">
     <string>Narzędzia</string>
    </property>
    <addaction name="actionZaznaczanie"/>
    <addaction name="actionPrzesuwanieWidoku"/>
    <addaction name="actionPrzesuwanie"/>
    <addaction name="actionUsun"/>
    <addaction name="separator"/>
    <addaction name="actionDodaj"/>
    <addaction name="actionEdycjaWierzcholkow"/>
    <addaction name="separator"/>
    <addaction name="actionDodajPunktRuchu"/>
    <addaction name="actionLaczeniePunktowRuchu"/>
    <addaction name="separator"/>
    <addaction name="actionUstawPunktPoczatkowy"/>
    <addaction name="actionDodajObcyPojazd"/>
   </widget>
   <addaction name="menuPlik"/>
   <addaction name="menuWidok"/>
   <addaction name="menuNarzedzia"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNowaPlansza"/>
   <addaction name="actionWczytaj"/>
   <addaction name="actionZapisz"/>
   <addaction name="separator"/>
   <addaction name="actionSpecyfikacja"/>
   <addaction name="separator"/>
   <addaction name="actionOryginalnyRozmiar"/>
   <addaction name="actionPrzybliz"/>
   <addaction name="actionOddal"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionZaznaczanie"/>
   <addaction name="actionPrzesuwanieWidoku"/>
   <addaction name="actionPrzesuwanie"/>
   <addaction name="actionUsun"/>
   <addaction name="actionDodaj"/>
   <addaction name="actionEdycjaWierzcholkow"/>
   <addaction name="actionDodajPunktRuchu"/>
   <addaction name="actionLaczeniePunktowRuchu"/>
   <addaction name="actionUstawPunktPoczatkowy"/>
   <addaction name="actionDodajObcyPojazd"/>
   <addaction name="actionUstawMeteLabiryntu"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>524287</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Drzewo elementów</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="margin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QTreeWidget" name="treeWidget">
       <property name="indentation">
        <number>16</number>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string>1</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>Plansza</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/ikony/map.png</normaloff>:/ikony/map.png</iconset>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget2">
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Drzewo punktów ruchu</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents2">
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="margin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QTreeWidget" name="treeWidget2">
       <property name="indentation">
        <number>16</number>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string>1</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>Plansza</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/ikony/map.png</normaloff>:/ikony/map.png</iconset>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionWczytaj">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ikony/folder.png</normaloff>:/ikony/folder.png</iconset>
   </property>
   <property name="text">
    <string>Wczytaj</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionWyjdz">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ikony/cancel.png</normaloff>:/ikony/cancel.png</iconset>
   </property>
   <property name="text">
    <string>Wyjdź</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionZapisz">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ikony/disk.png</normaloff>:/ikony/disk.png</iconset>
   </property>
   <property name="text">
    <string>Zapisz</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNowaPlansza">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ikony/page_white.png</normaloff>:/ikony/page_white.png</iconset>
   </property>
   <property name="text">
    <string>Nowa plansza</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionDodaj">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ikony/shape_square_add.png</normaloff>:/ikony/shape_square_add.png</iconset>
   </property>
   <property name="text">
    <string>Dodaj przeszkodę</string>
   </property>
   <property name="toolTip">
    <string>Dodaj przeszkodę</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </action>
  <action name="actionUsun">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ikony/delete.png</normaloff>:/ikony/delete.png</iconset>
   </property>
   <property name="text">
    <string>Usuń zaznaczone elementy</string>
   </property>
   <property name="toolTip">
    <string>Usuń zaznaczone elementy</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionPrzybliz">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ikony/zoom_in.png</normaloff>:/ikony/zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>Przybliż</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionOddal">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ikony/zoom_out.png</normaloff>:/ikony/zoom_out.png</iconset>
   </property>
   <property name="text">
    <string>Oddal</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionPrzesuwanie">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ikony/arrow_nsew.png</normaloff>:/ikony/arrow_nsew.png</iconset>
   </property>
   <property name="text">
    <string>Przesuwanie elementów</string>
   </property>
   <property name="toolTip">
    <string>Przesuwanie</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionEdycjaWierzcholkow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ikony/shape_handles.png</normaloff>:/ikony/shape_handles.png</iconset>
   </property>
   <property name="text">
    <string>Edycja wierzchołków</string>
   </property>
   <property name="toolTip">
    <string>Edycja wierzchołków</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+5</string>
   </property>
  </action>
  <action name="actionZaznaczanie">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ikony/cursor.png</normaloff>:/ikony/cursor.png</iconset>
   </property>
   <property name="text">
    <string>Zaznaczanie</string>
   </property>
   <property name="toolTip">
    <string>Zaznaczanie</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionPrzesuwanieWidoku">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ikony/hand.png</normaloff>:/ikony/hand.png</iconset>
   </property>
   <property name="text">
    <string>Przesuwanie widoku</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionZapiszJako">
   <property name="text">
    <string>Zapisz jako...</string>
   </property>
  </action>
  <action name="actionOryginalnyRozmiar">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ikony/zoom_actual.png</normaloff>:/ikony/zoom_actual.png</iconset>
   </property>
   <property name="text">
    <string>Oryginalny rozmiar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionDodajPunktRuchu">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ikony/flag_yellow.png</normaloff>:/ikony/flag_yellow.png</iconset>
   </property>
   <property name="text">
    <string>Dodaj punkt ruchu</string>
   </property>
   <property name="toolTip">
    <string>Dodaj punkt ruchu</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+6</string>
   </property>
  </action>
  <action name="actionLaczeniePunktowRuchu">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ikony/flag_connect.png</normaloff>:/ikony/flag_connect.png</iconset>
   </property>
   <property name="text">
    <string>Łączenie punktów ruchu</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+7</string>
   </property>
  </action>
  <action name="actionSpecyfikacja">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ikony/map.png</normaloff>:/ikony/map.png</iconset>
   </property>
   <property name="text">
    <string>Specyfikacja planszy</string>
   </property>
  </action>
  <action name="actionUstawPunktPoczatkowy">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ikony/joystick.png</normaloff>:/ikony/joystick.png</iconset>
   </property>
   <property name="text">
    <string>Ustaw punkt początkowy</string>
   </property>
   <property name="toolTip">
    <string>Ustaw punkt początkowy gracza</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+8</string>
   </property>
  </action>
  <action name="actionDodajObcyPojazd">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ikony/computer.png</normaloff>:/ikony/computer.png</iconset>
   </property>
   <property name="text">
    <string>Dodaj obcy pojazd</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+9</string>
   </property>
  </action>
  <action name="actionUstawMeteLabiryntu">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/ikony/flag_checked.png</normaloff>:/ikony/flag_checked.png</iconset>
   </property>
   <property name="text">
    <string>Ustaw metę labiryntu</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionWczytaj</sender>
   <signal>triggered()</signal>
   <receiver>OknoGlowne</receiver>
   <slot>wczytajPlansze()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNowaPlansza</sender>
   <signal>triggered()</signal>
   <receiver>OknoGlowne</receiver>
   <slot>nowaPlansza()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionWyjdz</sender>
   <signal>triggered()</signal>
   <receiver>OknoGlowne</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUsun</sender>
   <signal>triggered()</signal>
   <receiver>OknoGlowne</receiver>
   <slot>usunZaznaczony()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZapisz</sender>
   <signal>triggered()</signal>
   <receiver>OknoGlowne</receiver>
   <slot>zapiszPlansze()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZapiszJako</sender>
   <signal>triggered()</signal>
   <receiver>OknoGlowne</receiver>
   <slot>zapiszPlanszeJako()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPrzybliz</sender>
   <signal>triggered()</signal>
   <receiver>OknoGlowne</receiver>
   <slot>przyblizZoom()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOddal</sender>
   <signal>triggered()</signal>
   <receiver>OknoGlowne</receiver>
   <slot>oddalZoom()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPrzesuwanie</sender>
   <signal>triggered()</signal>
   <receiver>OknoGlowne</receiver>
   <slot>trybPrzesuwania()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEdycjaWierzcholkow</sender>
   <signal>triggered()</signal>
   <receiver>OknoGlowne</receiver>
   <slot>trybEdycjiWierzcholkow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZaznaczanie</sender>
   <signal>triggered()</signal>
   <receiver>OknoGlowne</receiver>
   <slot>trybZaznaczania()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPrzesuwanieWidoku</sender>
   <signal>triggered()</signal>
   <receiver>OknoGlowne</receiver>
   <slot>trybPrzesuwaniaWidoku()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOryginalnyRozmiar</sender>
   <signal>triggered()</signal>
   <receiver>OknoGlowne</receiver>
   <slot>zerujZoom()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDodaj</sender>
   <signal>triggered()</signal>
   <receiver>OknoGlowne</receiver>
   <slot>trybDodawaniaPrzeszkod()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLaczeniePunktowRuchu</sender>
   <signal>triggered()</signal>
   <receiver>OknoGlowne</receiver>
   <slot>trybLaczeniaWaypointow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDodajPunktRuchu</sender>
   <signal>triggered()</signal>
   <receiver>OknoGlowne</receiver>
   <slot>trybDodawaniaWaypointow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSpecyfikacja</sender>
   <signal>triggered()</signal>
   <receiver>OknoGlowne</receiver>
   <slot>edytujSpecyfikacje()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUstawPunktPoczatkowy</sender>
   <signal>triggered()</signal>
   <receiver>OknoGlowne</receiver>
   <slot>trybPozycjiGracza()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDodajObcyPojazd</sender>
   <signal>triggered()</signal>
   <receiver>OknoGlowne</receiver>
   <slot>trybObcegoPojazdu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUstawMeteLabiryntu</sender>
   <signal>triggered()</signal>
   <receiver>OknoGlowne</receiver>
   <slot>trybMety()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>wczytajPlansze()</slot>
  <slot>nowaPlansza()</slot>
  <slot>usunZaznaczony()</slot>
  <slot>zapiszPlansze()</slot>
  <slot>zapiszPlanszeJako()</slot>
  <slot>przyblizZoom()</slot>
  <slot>oddalZoom()</slot>
  <slot>trybPrzesuwania()</slot>
  <slot>trybEdycjiWierzcholkow()</slot>
  <slot>trybZaznaczania()</slot>
  <slot>trybPrzesuwaniaWidoku()</slot>
  <slot>zerujZoom()</slot>
  <slot>trybDodawaniaPrzeszkod()</slot>
  <slot>trybLaczeniaWaypointow()</slot>
  <slot>trybDodawaniaWaypointow()</slot>
  <slot>edytujSpecyfikacje()</slot>
  <slot>trybPozycjiGracza()</slot>
  <slot>trybObcegoPojazdu()</slot>
  <slot>trybMety()</slot>
 </slots>
</ui>
